var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -48,13 +48,20 @@ class Gromacs(CMakePackage):
             description='The build type to build',
             values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel',
                     'Reference', 'RelWithAssert', 'Profile'))
+    variant('simd', default='AUTO',
+            description='The SIMD instruction set to use',
+            values=('AUTO', 'None', 'SSE2', 'SSE4.1', 'AVX_128_FMA', 'AVX_256',
+                    'AVX2_128', 'AVX2_256', 'AVX_512', 'AVX_512_KNL',
+                    'IBM_QPX', 'Sparc64_HPC_ACE', 'IBM_VMX', 'IBM_VSX',
+                    'ARM_NEON', 'ARM_NEON_ASIMD'))
+    variant('rdtscp', default=True, description='Enable RDTSCP instruction usage')
 
     depends_on('mpi', when='+mpi')
     depends_on('plumed+mpi', when='+plumed+mpi')
     depends_on('plumed~mpi', when='+plumed~mpi')
     depends_on('fftw')
     depends_on('cmake@2.8.8:3.9.99', type='build')
```

Comment:
same change as below here?

Reply:
Sorry, I don't know what you are asking me here :-/

Reply:
One line below you removed the upper version bound on CMake (which should have been `:3:99.99`). Do you want to remove it here as well?
In both cases, you could also just update them from `:3:9.99` to `:3:99.99`

Reply:
I am am a big fan of adding upper bounds for dependencies when breaking changes are likely, e.g. for new major releases for semver projects, but you can decide on this.

Reply:
Sounds good. I changed it.